std_msgs/Float64[] landscape
---
geometry_msgs/Pose2D goal
std_msgs/Float64 pert_value
